README.md

```diff
@@ -8,6 +8,7 @@ Generate Backstage Catalog Information as part of the Quarkus build or the Quark
 
 - Generate the catalog-info.yaml for the Quarkus application
 - Command Line interface to install / uninstall and Component to Backstage
+- Generate Backstage Template from an existing Quarkus application
```

Comment:
Generate a Backstage Template for ....
Remark: You should here explain a bit what the template will do

Reply:
Doc updated with more details about the generated template. However, I respectfully disagree on the `s/from/for` thing. We are not generating a template `for` an application as we do for example with the `catalog-info.yaml`. We are using the application itself as input and thus `from` is more appropriate in this case.